src/makeRobot.js

```diff
@@ -39,6 +39,64 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    version,
+    wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    get info() {
+      return `name: ${this.name}, chip version: `
+      + `${this.version}, wheels: ${this.wheels}`;
+    },
+    set setX(n) {
+      this.coords.x = n;
+    },
+    set setY(n) {
+      this.coords.y = n;
+    },
+    evacuate() {
+      this.setX = 1400;
+      this.setY = 500;
+
+      return this;
+    },
+    goRight(n = 1) {
+      if (n > 0) {
+        this.setX = this.coords.x + n;
+      }
+
+      return this;
+    },
+    goBack(n = 1) {
```

Comment:
```suggestion
    goBack(step = 1) {
```
use more representative names